openpype/modules/deadline/plugins/publish/submit_publish_job.py

```diff
@@ -235,6 +235,8 @@ def _submit_deadline_post_job(self, instance, job, instances):
             if mongo_url:
                 environment["OPENPYPE_MONGO"] = mongo_url
 
+        priority = self.deadline_priority or instance.data.get("priority")
```

Comment:
This could potentially result in a `None` value - however I don't believe you can submit to Deadline with None value as Priority? If None is not a valid value for the submission we might need to do `instance.data.get("priority", 50)`.

Or if instead we believe the priority instance data is a fallback that should always exist I think we should rely on it with `instance.data["priority"]`?

Reply:
Used first variant, as it seems safer.